wahoomc/timings.py

```diff
@@ -0,0 +1,46 @@
+"""
+functions for timing sections of code, and output results in log lines
+"""
+#!/usr/bin/python
+
+# import official python packages
+import time
+import logging
+
+log = logging.getLogger('main-logger')
+
+
+class Timings:
+    """Class for starting the wall time, and logging when done"""
+    wall_time_start = None
+    wall_time_end = None
+
+    def start_wall_time(self):
+        """Start the wall time"""
+        self.wall_time_start = time.perf_counter()
+
+    def stop_wall_time(self):
+        """Stop the wall time"""
+        self.wall_time_end = time.perf_counter()
+
+    def stop_and_return(self, additional_info='') -> str:
+        """Stop and return text"""
+        self.stop_wall_time()
+        return self.get_text_summary(additional_info)
+
+    def stop_and_log(self, additional_info=''):
+        """Log the time taken"""
+        self.stop_wall_time()
+        self.log_duration(additional_info)
+
+    def log_duration(self,additional_info=''):
+        """Log the time taken"""
+        log.info(self.get_text_summary(additional_info))
```

Comment:
```suggestion
        log.info('-' * 80)
        log.info(self.get_text_summary(additional_info))
```

that should do the job in my opinion. `log_duration` is only called once so cutting down the method makes it clearer and identical to `stop_and_return` in my opinion

plus painting a "line" before the logging to be cosistent to the other parts of the log